### PR TITLE
email-alert-api AWS RDS

### PR DESCRIPTION
**Pull request by suthagarht**
- We are aiming to create a separate AWS RDS instance for
email-alert-api.

- This commit covers the components required to create a seperate
postgresql instance for email-alert-api.

https://trello.com/c/ljsMBGQ4/1419-split-email-alert-api-into-its-own-rds-instance

Solo: @suthagarht

### CODE REVIEW

File: terraform/projects/app-email-alert-api-postgresql/main.tf [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+variable "snapshot_identifier" {
+  type        = "string"
+  description = "Specifies whether or not to create the database from this snapshot"
+  default     = ""
+}
+
+variable "storage_size" {
+  type        = "string"
+  description = "Defines the AWS RDS storage capacity, in gigabytes."
+  default     = "60"
```
Comment (issyl0) [inline terraform/projects/app-email-alert-api-postgresql/main.tf]: I commented about this in https://github.com/alphagov/govuk-aws-data/pull/194/files/e76ceae4ebae5f54ac90a2f0941ae087010916d0#diff-918c78288dbda688eddedd845898f9aa

File: terraform/projects/infra-security-groups/email-alert-api-postgresql.tf [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  protocol  = "tcp"
+
+  # Which security group is the rule assigned to
+  security_group_id = "${aws_security_group.email-alert-api-postgresql-primary.id}"
+
+  # Which security group can use this rule
+  source_security_group_id = "${aws_security_group.email-alert-api-db-admin.id}"
+}
+
+resource "aws_security_group_rule" "email-alert-api-postgresql-standby_ingress_bouncer_postgres" {
```
Comment (issyl0) [inline terraform/projects/infra-security-groups/email-alert-api-postgresql.tf]: Bouncer doesn't talk to Email Alert API. This has been left over from Transition. :-)

File: terraform/projects/app-email-alert-api-postgresql/main.tf [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  region  = "${var.aws_region}"
+  version = "1.14.0"
+}
+
+module "email-alert-api-postgresql-primary_rds_instance" {
+  source = "../../modules/aws/rds_instance"
+
+  name                = "${var.stackname}-email-alert-api-postgresql-primary"
+  engine_name         = "postgres"
+  engine_version      = "9.6"
```
Comment (issyl0) [inline terraform/projects/app-email-alert-api-postgresql/main.tf]: Have we upgraded to 9.6 everywhere now? I thought some things were still on 9.3?

Comment (suthagarht) [reply terraform/projects/app-email-alert-api-postgresql/main.tf]:
```./app-warehouse/main.tf:  engine_version      = "9.6"
./app-postgresql/main.tf:  engine_version      = "9.6"
./app-transition-postgresql/main.tf:  engine_version      = "9.6"
```

Seems to be 9.6 if I am not missing something. Please feel free to correct me.

Comment (issyl0) [reply terraform/projects/app-email-alert-api-postgresql/main.tf]: You're right. I was thinking of Carrenza.

**Pull request merged**